.github/workflows/publish_docker_images_on_push.yml

```diff
@@ -3,18 +3,17 @@ name: Build Docker Image On Push (Both Tags and Branches)
 on:
   push:
     branches:
-      - '**'
+      - "**"
     tags:
-      - '**'
+      - "**"
       -
 jobs:
-
   build_and_publish_images:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        image_type: [latest,alpine,postgres_12 ]
+        image_type: [latest, alpine, postgres_12, nightly]
```

Comment:
IMO, this would be cleaner if separated into 4 new lines.

Reply:
Fixed

Reply:
It appears as though you forgot to push again @gurkanindibay

Reply:
Before you suggested putting empty spaces between items. I applied that
Now I applied list structure